packages/ui/src/components/PropertiesModal/PropertiesTabs.tsx

```diff
@@ -0,0 +1,36 @@
+import { Tab, Tabs } from '@patternfly/react-core';
+import React, { FunctionComponent, useState } from 'react';
+import { IPropertiesTab } from './PropertiesModal.models';
+import { PropertiesTable } from './PropertiesTable';
+
+interface IPropertiesTabsProps {
+  tabs: IPropertiesTab[];
+}
+
+export const PropertiesTabs: FunctionComponent<IPropertiesTabsProps> = (props) => {
+  const [activeTabKey, setActiveTabKey] = useState<string | number>(0);
+  const handleTabClick = (
+    _event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabIndex: string | number,
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  return (
+    <Tabs
+      activeKey={activeTabKey}
+      onSelect={handleTabClick}
+      aria-label="Properties tabs"
+      isBox={true}
```

Comment:
This is an alternative.

```suggestion
      isBox
```

If there are no pending changes from your side, please feel free to postpone it for the next pull request :)

Reply:
thx, i will update it in the next pr